### ISSUE TITLE
Tutorial feedback

### ISSUE THREAD

**Issue by mrocklin** [holoviz/holoviz#97]
I helped @jbednar deliver a tutorial to a group that was only semi-literate in Python.  Here are a few bits of feedback:

1.  [x] Use a separate conda environment.  Many people downloaded anaconda a long time ago and now have conflicting versions, particularly with Jupyter/Tornado
2.  [ ] Consider building a single binder environment that users can use if their environment doesn't work
3.  [x] Remove exclamation marks in the setup notebook.  People were copy-pasting this into their command prompt (windows users especially)
4.  [ ] The conda environment takes several minutes to resolve.  Pushed upstream here: https://github.com/conda/conda/issues/7239#issuecomment-421014770

**Comment by ceball** [comment]
Thanks!

> 1. Use a separate conda environment

I think we concluded that ourselves before, and I thought we'd already changed it. But I see we say creating an environment is "optional" at http://pyviz.org/installation.html and we don't mention it all on the various tutorial index pages (e.g. http://pyviz.org/tutorial/00_Setup.html). I vote that we should just tell people to do it, consistently. I'm going to change the text unless anyone objects...

> 2. Consider building a single binder environment that users can use if their environment doesn't work

Yes, we should do this; I've made #98.

> 3. Remove exclamation marks in the setup notebook

I agree. Again, I'll change that unless anyone objects...

> 4. The conda environment takes several minutes to resolve. Pushed upstream

Thanks for taking the time to do that.
